pkg/sds/auth/universal/authenticator.go

```diff
@@ -18,34 +20,66 @@ func NewAuthenticator(issuer builtin_issuer.DataplaneTokenIssuer, dataplaneResol
 	}
 }
 
+// universalAuthenticator defines authentication for Dataplane Tokens
+// All fields in token are optional, so we only validate data that is available in token. This way you can pick your level of security.
+// Generate token for mesh+name for maximum security.
+// Generate token for mesh+tags(ex. kuma.io/service) so you can reuse the token for many instances of the same service.
+// Generate token for mesh if you trust the scope of the mesh.
+//
+// If you generate token bound to tags all tags values have to match the dataplane, so for example if you have a Dataplane
+// with inbounds: 1) kuma.io/service:web 2) kuma.io/service:web-api, you need token for both values kuma.io/service=web,web-api
```

Comment:
Interesting, can I do "wildcards" here?

Reply:
Wildcard in the form of `kuma.io/service: *` does not make sense. You can just omit the tag.
Wildcard in the form of `kuma.io/service: web*` makes sense, but I'm hesitant to introduce it now. I'd introduce it if there is a real need for it.